### DISCUSSION TITLE
Source definition

### DISCUSSION THREAD

**Discussion by yoongschoi** [nrc-cnrc/EGSnrc#1026] [Questions] [answered]
I was trying to place an isotropic source (Cs-137, 0.662 MeV) right underneath the geometry but it doesn't work. What did I do wrong with the source definition in this code?

#################################################
#################################################

:start source definition:
  
 :start source:
   library = egs_isotropic_source
   name = source
   position = 25 25 0 
   
   :start spectrum:
     type = monoenergetic
     energy = 0.662
   :stop spectrum:
   
   charge = 0
 :stop source:

 simulation source = source

:stop source definition:
```

**Comment by mxxo** [comment, marked as answer]
I can't confirm with `egs_view` right now but it might be that you're missing the `shape` input of the isotropic source? Try defining your source with a point shape: 

```ruby
:start shape:
    type = point
    position = 25 25 0 
:stop shape:
```

I believe if you run egs_view from the terminal, you should be able to see any errors printed upon loading the file.

**Reply by yoongschoi** [reply]
Yeah you're right, I didn't know I had to set shape. Thanks a lot!
